packages/edit-navigation/src/components/menus-editor/index.js

```diff
@@ -15,14 +19,16 @@ export default function MenusEditor( { blockEditorSettings } ) {
 	const menus = useSelect( ( select ) => select( 'core' ).getMenus() );
 
 	const [ menuId, setMenuId ] = useState( 0 );
+	const [ stateMenus, setStateMenus ] = useState( null );
 
 	useEffect( () => {
 		if ( menus?.length ) {
```

Comment:
If I have no menus, the spinner will keep on spinning forever.

Reply:
The same is true if the request to load the menus errors out

Reply:
To be fair, this isn't specific to this PR - it also happens without it.

Reply:
Yes, not ideal. However, considering we'd need to decide how to treat this visually, we can open a separate issue for better results.